application/controllers/handler.js

```diff
@@ -66,55 +67,36 @@ let handlers = module.exports = {
       if (fs.existsSync(filePath))
         return response(toReturn);
 
+      if (!fs.existsSync(folder))
+        fs.mkdirSync(folder);
+
       html = applyThemeToSlideHTML(html, theme);
 
       let document = cheerio.load(html);
-      let pptxheight = 0, pptxwidth = 0;
+      let width = 0, height = 0;
       try {
-        pptxwidth = document('div[class=pptx2html]').css().width.replace('px', '');
-        pptxheight = document('div[class=pptx2html]').css().height.replace('px', '');
+        width = document('div[class=pptx2html]').css().width.replace('px', '');
+        height = document('div[class=pptx2html]').css().height.replace('px', '');
       } catch (e) {
         //There's probably no css in the slide
       }
-      pptxwidth = pptxwidth ? pptxwidth : 0;
-      pptxheight = pptxheight ? pptxheight : 0;
-      let width = 0;
-      let height = 0;
-      if (pptxwidth !== 0 && pptxheight !== 0) {
-        width = pptxwidth;
-        height = pptxheight;
-      } else {
-        width = 'all';
-        height = 'all';
+      width = width ? width : 0;
+      height = height ? height : 0;
+      if (width === 0 || height === 0) {
+        width = '1920';
```

Comment:
I would set this to 960 by 720 - which powerpoint default and I guess more often used.. Or is this specifically set because 960*729 is detected and 1920*1080 is often not detected?

Reply:
This dictates the virtual browser window size, of which a screenshot is captured. As slides should be responsive, 960x720 should show no difference than 1920x1080 (both are 16:9). This is also just the fallback value, in case there is no size in the slide itself.